package.json

```diff
@@ -1,20 +1,26 @@
 {
-  "name": "websterParser",
-  "version": "1.0.0",
-  "description": "Parses the Webster Unabridged Dictionary from Project Guttenberg into a more usable format",
+  "private": true,
   "main": "index.js",
-  "author": "Leo Koppelkamm",
-  "license": "ISC",
```

Comment:
Probably good to leave the `name`, `version`, etc. in here, since it’s useful metadata (even if this is not published as a package like most modules).

Reply:
I removed it because all the metadata is visible elsewhere and just clutters up package.json.